examples/first-project/networks.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  networks: {
+    local: {
+      protocol: 'http',
+      host: 'localhost',
+      port: 8545,
+      gas: 5000000,
+      gasPrice: 5e9,
+      // eslint-disable-next-line @typescript-eslint/camelcase
+      network_id: '*',
+    },
+    test: {
+      protocol: 'http',
+      host: 'localhost',
+      port: 9555,
+      gas: 5000000,
+      gasPrice: 5e9,
+      // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
is there any way of blacklisting the `networks.js`/`truffle.js` files for the linter so we don't have to add this line?

Reply:
ha, btw: if I'm not wrong, the `network.js` file also understands `camelCase` so seting it as `networkId` can do the trick.